test/MINLPTests/run_minlptests.jl

```diff
@@ -0,0 +1,24 @@
+using MINLPTests, JuMP, SCIP, Test
+
+const OPTIMIZER = JuMP.with_optimizer(SCIP.Optimizer, display_verblevel=0)
+const OBJTOL = 1e-5
+const PRIMALTOL = 1e-5
+const DUALTOL = NaN  # to disable the query
+
+@testset "MINLPTests" begin
+    @testset "nlp_005_010" begin
+        MINLPTests.nlp_005_010(OPTIMIZER, OBJTOL, PRIMALTOL, DUALTOL;
+                               termination_target = MOI.OPTIMAL)
+    end
+
+    @testset "nlp_005_011" begin
+        MINLPTests.nlp_005_010(OPTIMIZER, OBJTOL, PRIMALTOL, DUALTOL;
```

Comment:
It looks like this should be `nlp_005_011`?

Reply:
Yes, thanks. There are also more to come from the `nlp_cvx` set...

Reply:
I now added all tests that I think can be supported now. For one test, SCIP had a wrong solution, which I will try to investigate upstream.